style.css

```diff
@@ -127,3 +127,33 @@ nav .btn {
   text-align: center;
   max-width: none;
 }
+
+.partners-text {
+  display: inline-flex;
+  font-size: 12px;
+  line-height: 160%;
+  text-decoration: none;
+  font-weight: 600;
+  border: 1px solid transparent;
+  border-radius: 4px;
```

Comment:
If the border is transparent, you won't even see it. So is there any advantage of having these rules at all?